download_jartic_archives.py

```
@@ -0,0 +1,127 @@
+    months_to_download = []
+    current = datetime(start_year, start_month, 1)
+    end = datetime(end_year, end_month, 1)
+    
+    while current <= end:
+        months_to_download.append((current.year, current.month))
+        if current.month == 12:
+            current = datetime(current.year + 1, 1, 1)
+        else:
+            current = datetime(current.year, current.month + 1, 1)
+    
+    print(f"\n🚗 JARTIC Archive Downloader")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)